packages/react-composites/src/composites/CallComposite/Strings.tsx

```diff
@@ -763,4 +763,29 @@ export interface CallCompositeStrings {
    * String for more details when the call times out because the remote user does not answer
    */
   callTimeoutDetails?: string;
+  /* @conditional-compile-remove(dtmf-dialer) */
+  /**
+   * Label for the control bar button to show the dtmf dialer when the more button is disabled
+   */
+  dtmfDialerButtonLabel?: string;
```

Comment:
Where is this used?

Reply:
this is to be used on the button when its rendered in place of the more button. will make sure it is used. guess I forgot!